preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -965,6 +988,29 @@ def run(config: Config) -> None:
                     f"Processing failed. Traceback : {e}. Unprocessed data: {unprocessed}"
                 )
                 continue
+
+            for processed_entry in processed:
+                group_id = int(str(processed_entry.data.metadata["groupId"]))
+                del processed_entry.data.metadata["groupId"]  # Remove groupId after extraction
+                file_content = create_flatfile(
+                    config,
+                    processed_entry.accession,
+                    processed_entry.version,
+                    processed_entry.data.metadata,
+                    processed_entry.data.unalignedNucleotideSequences,
+                    processed_entry.data.annotations,
+                )
+                processed_entry.data.annotations = None  # remove it so it's not submitted
+                upload_info = request_upload(group_id, 1, config)[0]
+                file_id = upload_info.fileId
+                url = upload_info.url
+                upload_string_to_presigned_url_zipped(file_content, url)
+                processed_entry.data.files = {
+                    "annotations": [
+                        FileIdAndName(fileId=file_id, name="sequences.embl.gz")
```

Comment:
Do we want to upload as .gz? Is that because webin expects that? It might be nicer to have it uncompressed maybe for people to just view directly after download.

Reply:
The backend doesn't do anything to the files that get uploaded, it's also uploaded with a presigned URL directly to S3, so no additional processing there. Then I'll go ahead and not compress it, and instead compress it before it gets submitted!

Reply:
Generally I'm happy with that approach - I just wonder will it increase the file upload time? We should probably check if it starts taking much longer (but we should see this on the preview without custom measurements if it is really a problem)

Reply:
I've changed this, I'm marking this as resolved, haven't noticed a slower upload time yet